File: Cheer_Creator/Cheer_Creator.cpp
#include <iostream>
using namespace std;

int main() {
     
     int yards;
     cin >> yards;
     if(yards > 10)
     cout<< "High Five";
     else if(yards < 1)
     cout<< "shh";
     else { 
     for(int x=0;x<yards;x++)
     cout<< "Ra!";
      }
    return 0;
}

File: Cheer_Creator/Cheer_Creator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int yards = Int32.Parse(Console.ReadLine());
            if(yards > 10)
             Console.WriteLine("High Five");
             else if(yards < 1)
             Console.WriteLine("shh");
             else
             {
                 for(int i=0; i<yards;i++)
                 Console.WriteLine("Ra!");
             }
        }
    }
}
